dpnp/dpnp_iface_indexing.py

```diff
@@ -194,6 +189,15 @@ def diag_indices(n, ndim=2):
 
     For full documentation refer to :obj:`numpy.diag_indices`.
```

Comment:
need to add CFD keywords (`device`, `sycl_queu`, `usm_type`), like it was did for `dpnp.indices`